plugins/modules/vmware_host_snmp.py

```diff
@@ -73,6 +73,14 @@
     type: str
     choices: [ debug, info, warning, error ]
     default: info
+  sys_contact:
+    description:
+        - System contact who manages the system.
+    type: str
```

Comment:
Please create a [changelog fragment](https://docs.ansible.com/ansible/latest/community/development_process.html#creating-a-changelog-fragment). See [this fragment](https://github.com/ansible/ansible/blob/c2f528b7686c9573aa8493357219e0bbe4416dd3/changelogs/fragments/reboot-change-default-boot-command.yaml) as an example.